[2025-09-06 11:23:56] [normal_8] ERROR: [normal_8] An error occurred: [Errno 111] Connection refused
